tests/end-to-end/upgrade/README.md

```diff
@@ -55,6 +55,9 @@ go run main.go --action prepareData --verbose
 
 ## Development
 
+>**NOTE:** The following approach for adding new tests to the upgrade scenario is not needed anymore. To prepare data during the preparation phase, create a new TestDefinition that executes your preparation code and add the label `kyma-project.io/before-upgrade=true`. To execute evaluation code during the evaluation phase, create a new TestDefinition with the label `kyma-project.io/after-upgrade=true` 
```

Comment:
This section is about how to implement new upgrade test - for example to a new domain. I don't understand why you wrote the first sentence in the note.  This section describes how to implement the test, the interface which must be implemented, where the code should be created (which folder).

Reply:
I wrote this, because in order to implement a new upgrade test you don't have to shoehorn it into the existing upgrade test anymore. as long as you label the testdefinition correctly you are totally free to implement the test however you like. This makes it easier to introduce your own dependencies as you are not longer limited by dependencies of other tests (eg knative eventing 0.12 limits the version of the kubernetes client)